### PR TITLE
feat(cli): Add initial ORT Server CLI implementation

### PR DESCRIPTION
**Pull request by bs-ondem**
This commit introduces the new module `cli` for the ORT Server, intended to contain all upcoming implementations related to the CLI.

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -25,6 +25,7 @@ rootProject.name = "ort-server"
 
 include(":api:v1:mapping")
 include(":api:v1:model")
+include("cli")
```
Comment (sschuberth) [inline settings.gradle.kts:28]: This should have a leading ":" to make it "absolute".

**Comment by sschuberth** [comment]
> all upcoming implementations related to the CLI.

Interesting, I wasn't aware such a thing being planned. Can you share more about the use-case and planned commands?

**Comment by bs-ondem** [comment]
The CLI will simplify the integration of the ORT Server in a CI/CD environment for our customers, especially starting an ORT run in a blocking mode. To replace our proprietary solution with the ORT Server CLI, we need the following features:
- Start an ORT run
- Get details about an ORT run
- Download reports
- Download logs

File: cli/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+jib {
+    from.image = "eclipse-temurin:${libs.versions.eclipseTemurin.get()}"
+    to.image = "${dockerImagePrefix}ort-server-cli:$dockerImageTag"
+
+    container {
+        mainClass = "org.eclipse.apoapsis.ortserver.cli.OrtServerMainKt"
+        creationTime.set("USE_CURRENT_TIMESTAMP")
+    }
+}
```
Comment (sschuberth) [inline cli/build.gradle.kts:58]: Could the CLI be included in any Docker image? So is JIB needed after all?

Comment (bs-ondem) [reply cli/build.gradle.kts:58]: Basically yes, but I don't get your point to remove the Jib configuration? With the Docker image the CLI can be used from every environment (of course with Docker).

Comment (oheger-bosch) [reply cli/build.gradle.kts:58]: For distribution the container image is rather convenient. All CI platforms support a way to directly execute containers. A native executable would have to be installed using an alternative mechanism.
But of course, the container image does not have to be the only artifact generated by this project. The plan is further to have a client library that can be used to interact with ORT Server programmatically. The CLI is just one client of this library.

**Review by oheger-bosch — APPROVED** [review]

File: cli/build.gradle.kts (hunk 2 above)
Comment (sschuberth) [reply cli/build.gradle.kts:58]: > The plan is further to have a client library that can be used to interact with ORT Server programmatically. The CLI is just one client of this library.

Yeah, I guess what confused me is that this CLI stub is added even before that required client library is ready.

Comment (bs-ondem) [reply cli/build.gradle.kts:58]: Client library will be added in the upcoming PRs. The reason to add the CLI first is just due to the order of the issues in our internal board.

**Pull request merged**